_posts/2017-05-17-forbes.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Forbes
+sourceURL: https://www.youtube.com/watch?v=JmC0xkCMFCE
+image: /images/forbes.png
+tags:
+- News
+- Notifications
+- Service Worker
+- Offline
+
+---
+
+Forbes redesigned their mobile experience as a PWA resulting in **43% increase** in sessions per user, ad viewability up **20%**, and **100% more engagement**.
```

Comment:
> I should also point out that at some point it looks like we started bolding more than the figures (so 43% increase instead of 43%). 

Yeah, started noticing this as well. Consistency would be appreciated. 😁 

> I prefer only the company/app names and figures themselves bolded but I'm open to alternate opinions.

I also prefer only the company/app name + figures themselves bolded, but I'm not going to throw a fit if we don't roll with this. 😉